mono/tests/libtest.c

```diff
@@ -5630,8 +5630,7 @@ mono_test_marshal_return_single_double_struct (void)
 	return res;
 }
 
-
-#ifndef TARGET_X86
+#ifndef HOST_X86
```

Comment:
Need to undo this line, in this PR, and deal with it separately. This single line change breaks tests.